### PR TITLE
(Add) feature directions, currently hard coded.

### PR DESCRIPTION
**Pull request by tenhold**
Most of the files that were changed were fixing merge issues. Directions is commented out right now because of the API issue.

### CODE REVIEW

File: server/routes/customer.js [hunk 1]
```diff
@@ -27,8 +27,10 @@ const googleAuth = async (authToken) => {
     idToken: authToken,
     audience: '933644302187-agamsig0qalm5oi4fd44v11hfffpchs8.apps.googleusercontent.com',
   });
+  console.log('ticket', ticket);
```
Comment (jstrzesz) [inline server/routes/customer.js]: remove console.logs

File: server/routes/customer.js [hunk 2]
```diff
@@ -27,8 +27,10 @@ const googleAuth = async (authToken) => {
...
   const payload = ticket.getPayload();
-  console.info(`USER ${payload.name} VERIFIED`);
+  console.log('payload', payload)
```
Comment (jstrzesz) [inline server/routes/customer.js]: remove console.logs

File: server/routes/customer.js [hunk 3]
```diff
@@ -48,30 +50,30 @@ customerRouter.get('/gId/:gId', (req, res) => {
...
       } else {
-        res.send('empty');
+        res.send('empty')
       }
     })
     .catch((err) => {
-      console.warn('ERROR IN CHECK FOR ALL CUSTOMERS');
-    });
-});
+      console.error('ERROR IN CHECK FOR ALL CUSTOMERS')
```
Comment (jstrzesz) [inline server/routes/customer.js]: change to console.warn

File: server/routes/customer.js [hunk 4]
```diff
@@ -48,30 +50,30 @@ customerRouter.get('/gId/:gId', (req, res) => {
...
+    }
   })
     .then((customers) => {
       res.send(customers);
     })
     .catch((err) => {
-      console.warn('ERROR IN CHECK FOR CUSTOMER BY GoogleID');
-    });
-});
+      console.error('ERROR IN CHECK FOR CUSTOMER BY GoogleID')
```
Comment (jstrzesz) [inline server/routes/customer.js]: change to console.warn

File: src/components/Customer/Create/Map2.jsx [hunk 5]
```diff
@@ -98,38 +121,39 @@ const MapContainer = ({ setMapLatLng, gId }) => {
...
 
-  const handleMarkerClick = (e) => {
+  const handleMarkerClick = () => {
     setClick(!click);
   };
 
   const getMyLocation = ({ latitude, longitude }) => {
     axios.put(`/db/maps/${gId}`, { latitude, longitude })
-      .then((data) => console.info('maps put success'))
+      .then(() => console.info('maps put success'))
```
Comment (jstrzesz) [inline src/components/Customer/Create/Map2.jsx:156]: good

File: src/components/Customer/Create/Map2.jsx [hunk 6]
```diff
@@ -153,31 +177,50 @@ const MapContainer = ({ setMapLatLng, gId }) => {
...
-      />
-      {click
-        ? (
-          <BarInfo
-            placeInfo={placeInfo}
-            searchMarker={searchMarker}
-          />
-        )
-        : null}
+    <div style={{ align: 'center' }}>
```
Comment (jstrzesz) [inline src/components/Customer/Create/Map2.jsx]: remove in-line styling, move to a css file

File: src/components/Customer/Directions/Directions.jsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  const directionsCallback = (response) => {
+
+    if (response !== null) {
+      response.status === 'OK' ? setResponse(() => response)
+        : console.log('direction response: ', response);
+    }
+  };
+
+  return (
+    <>
```
Comment (jstrzesz) [inline src/components/Customer/Directions/Directions.jsx]: no empty tags, either make them divs or remove them

File: src/components/Customer/Directions/Directions.jsx [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+        options={{
+          origin: origin,
+          destination: destination,
+          travelMode: 'DRIVING'
+        }}
+        callback={directionsCallback}
+      />
+      {response && <DirectionsRenderer
+        options={{ directions: response }}
+        onLoad={(dirs) => console.log('directions: ', dirs)}
```
Comment (jstrzesz) [inline src/components/Customer/Directions/Directions.jsx]: console.info

File: src/components/Customer/Profile/CustomerProfile.jsx [hunk 9]
```diff
@@ -11,17 +11,20 @@ const CustomerProfile = ({ setViewValue, gId }) => {
...
     fetch(`/db/customer/gId/${gId}`, {
       method: 'GET',
     })
-      .then((response) => response.json())
-      .then((data) => {
-        setData(data);
-        setCustomerId(data.id);
+      .then(response => response.json())
+      .then(data => {
+        console.log("PROFILE DATA", data)
```
Comment (jstrzesz) [inline src/components/Customer/Profile/CustomerProfile.jsx]: remove console.log

File: src/components/Customer/Profile/CustomerProfile.jsx [hunk 10]
```diff
@@ -11,17 +11,20 @@ const CustomerProfile = ({ setViewValue, gId }) => {
   const [customerId, setCustomerId] = useState(1);
   const [view, setView] = useState('Home');
   const [data, setData] = useState({
-    user_name: '',
+    user_name: ''
   });
 
   useEffect(() => {
+    // console.log("customer profile", gId)
     fetch(`/db/customer/gId/${gId}`, {
```
Comment (jstrzesz) [inline src/components/Customer/Profile/CustomerProfile.jsx:19]: move this function to its own method and then call it inside of useEffect

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

File: src/components/Customer/Profile/CustomerProfile.jsx (hunk 10 above)
Comment (tenhold) [reply src/components/Customer/Profile/CustomerProfile.jsx:19]: I'm not sure who's working on this code.

Comment (jstrzesz) [reply src/components/Customer/Profile/CustomerProfile.jsx:19]: ok, if its not your code, then don't touch it for now, but let's make a note about it and put a refactor ticket in the backlog

**Review by lschwall — APPROVED** [review]

**Review by Bri-B — APPROVED** [review]

**Pull request closed without merging**